pkg/lib/authenticationflow/declarative/data_force_change_password.go

```diff
@@ -0,0 +1,18 @@
+package declarative
+
+import authflow "github.com/authgear/authgear-server/pkg/lib/authenticationflow"
+
+type ForceChangePasswordData struct {
+	TypedData
+	PasswordPolicy *PasswordPolicy       `json:"password_policy,omitempty"`
+	Reason         *PasswordChangeReason `json:"reason,omitempty"`
```

Comment:
Rename this from `reason` to `force_change_reason`

So it now reads `force_change_reason: policy` and `force_change_reason: expiry`.